### PR TITLE
Add blog post on "How to Configure FerretDB to work on Percona Distribution for PostgreSQL"

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+----------+-----------------------------+-------+----------
+ ferretdb | _ferretdb_database_metadata | table | username
+ ferretdb | test_afd071e5               | table | username
+(2 rows)
+```
+
+We can see two tables: one containing the metadata for the database and the other containing the `test` collection we created on Studio 3T.
+Let's view the table content in PostgreSQL:
+
+```sql
```
Comment (AlekSi) [inline website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md:276]: https://github.com/FerretDB/FerretDB/blob/eb6170cb3fb51f0e007b0a8ed5436a3474dbc5d4/website/docs/contributing/writing-guide.md?plain=1#L109

File: website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+ferretdb=# table test_afd071e5;
+```
+
+```sh
```
Comment (AlekSi) [inline website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md:280]: https://github.com/FerretDB/FerretDB/blob/eb6170cb3fb51f0e007b0a8ed5436a3474dbc5d4/website/docs/contributing/writing-guide.md?plain=1#L109

**Comment by Fashander** [comment]
@AlekSi @ptrfarkas PTAL

File: website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+<!--truncate-->
+
+That's exactly what you get when you configure FerretDB for your Percona Distribution for PostgreSQL!
+
+In this article, we'll guide you through the advantages of using FerretDB and how you can configure it to work natively on Percona Distribution for PostgreSQL and explore how FerretDB's stored data appear in PostgreSQL.
+
+Before we go into the technical details, let's find out what these two solutions are all about.
+
+## What is Percona Distrubution of PostgreSQL?
```
Comment (AlekSi) [inline website/blog/2023-06-26-configure-ferretdb-work-percona-distribution-postgresql.md:25]:
```suggestion
## What is Percona Distribution for PostgreSQL?
```

@Fashander Please enable spellchecker in you editor

File: website/static/img/blog/percona-ferretdb.png
Comment (AlekSi) [inline website/static/img/blog/percona-ferretdb.png:1]: That image uses a wrong product name

**Pull request merged**